Dockerfile-acceptance

```diff
@@ -1,8 +1,9 @@
-FROM quay.io/ukhomeofficedigital/nodejs-base:v8
+FROM node:8
 
 WORKDIR /test
 
 COPY package.json package.json
+COPY package-lock.json package-lock.json
```

Comment:
There's no value in doing this if you still run `npm install` and not `npm ci`. 

👇